tests/tasks/restore_state.yml

```diff
@@ -0,0 +1,24 @@
+---
+- name: Remove /etc/sysconfig/network if there was no one
+  tags:
+    - 'tests::cleanup'
+  file:
+    path: /etc/sysconfig/network
+    state: absent
+  when:
+    - etc_sysconfig_network_stat is defined
+    - not etc_sysconfig_network_stat.stat.exists
+
+- name: Restore services
+  import_tasks: restore_services_state.yml
+
+- name: reload NetworkManager
```

Comment:
Is it a problem when NetworkManager or other services are installed during the test and not removed/disabled afterwards?